src/Http/Routing/src/ShortCircuit/RouteShortCircuitEndpointRouteBuilderExtensions.cs

```diff
@@ -34,7 +34,9 @@ public static IEndpointConventionBuilder MapShortCircuit(this IEndpointRouteBuil
             {
                 route = $"{routePrefix}/{{**catchall}}";
             }
-            group.Map(routePrefix, _shortCircuitDelegate).ShortCircuit(statusCode);
+            group.Map(routePrefix, _shortCircuitDelegate)
+                .ShortCircuit(statusCode)
+                .Add(endpoint => ((RouteEndpoint)endpoint).Order = 0);
```

Comment:
This is the default order.

To give it a lower priority you need to give it a higher value. For example, our fallback APIs use `int.MaxValue`.
 
An alternative is for these APIs to build on top of `MapFallback` and just replace the delegate in a convention.

Reply:
We should also give this a display name. See https://github.com/dotnet/aspnetcore/blob/bec278eabea54f63da15e10e654bdfa4168a2479/src/Http/Routing/src/Builder/FallbackEndpointRouteBuilderExtensions.cs#L68-L80

Reply:
I forgot to save the file before committing. This won't even compile. my bad!! 🤦‍♂️
Ok I will change the method and also add some functional tests for these extensions methods.